nifi-toolkit/nifi-toolkit-admin/src/main/groovy/org/apache/nifi/toolkit/admin/client/NiFiClientFactory.groovy

```diff
@@ -17,38 +17,19 @@
 package org.apache.nifi.toolkit.admin.client
 
 import org.apache.commons.lang3.StringUtils
-import org.apache.nifi.security.util.CertificateUtils
+import org.apache.http.conn.ssl.DefaultHostnameVerifier
 import org.apache.nifi.util.NiFiProperties
-import org.slf4j.Logger
-import org.slf4j.LoggerFactory
-import org.apache.http.conn.ssl.DefaultHostnameVerifier;
 
-
-
-import javax.naming.ldap.LdapName
-import javax.naming.ldap.Rdn
-import javax.net.ssl.HostnameVerifier
 import javax.net.ssl.KeyManagerFactory
 import javax.net.ssl.SSLContext
-import javax.net.ssl.SSLPeerUnverifiedException
-import javax.net.ssl.SSLSession
 import javax.net.ssl.TrustManagerFactory
 import javax.ws.rs.client.Client
 import javax.ws.rs.client.ClientBuilder
-import java.security.KeyManagementException
-import java.security.KeyStore
-import java.security.KeyStoreException
-import java.security.NoSuchAlgorithmException
-import java.security.SecureRandom
-import java.security.UnrecoverableKeyException
-import java.security.cert.Certificate
+import java.security.*
 import java.security.cert.CertificateException
-import java.security.cert.CertificateParsingException
-import java.security.cert.X509Certificate
 
 class NiFiClientFactory implements ClientFactory{
 
-    private static final Logger logger = LoggerFactory.getLogger(NiFiClientFactory.class)
```

Comment:
Why get rid of the logger?

Reply:
It was unused in this class after removing the NiFiHostnameVerifier

Reply:
Ok, figured that; it's good practice to have it there anyway so that it's consistent across classes and the use stays aligned with the pattern we've established.